patches/server/0067-Port-LazyDFU.patch

```diff
@@ -0,0 +1,49 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Andrew Steinborn <git@steinborn.me>
+Date: Tue, 2 Feb 2021 20:59:35 -0500
+Subject: [PATCH] Port LazyDFU
+
+
+diff --git a/src/main/java/me/steinborn/lazydfu/mod/LazyDataFixerBuilder.java b/src/main/java/me/steinborn/lazydfu/mod/LazyDataFixerBuilder.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..c3744057877347c715ed944ec8ea557e28ac936f
+--- /dev/null
++++ b/src/main/java/me/steinborn/lazydfu/mod/LazyDataFixerBuilder.java
+@@ -0,0 +1,22 @@
++package me.steinborn.lazydfu.mod;
++
++import com.mojang.datafixers.*;
++
++import java.util.concurrent.Executor;
++
++/**
++ * This version of {@code DataFixerBuilder} does not immediately initialize rules.
++ */
++public class LazyDataFixerBuilder extends DataFixerBuilder {
++
++    private static final Executor NO_OP_EXECUTOR = command -> {};
++
++    public LazyDataFixerBuilder(int dataVersion) {
++        super(dataVersion);
++    }
++
++    @Override
++    public DataFixer build(Executor executor) {
++        return super.build(NO_OP_EXECUTOR);
++    }
++}
+\ No newline at end of file
+diff --git a/src/main/java/net/minecraft/server/DataConverterRegistry.java b/src/main/java/net/minecraft/server/DataConverterRegistry.java
+index b149e32aac86ce2f7521958fa7394e4f5b852174..aae968e8659baf2b14831df9ea8cae3eff204cf5 100644
+--- a/src/main/java/net/minecraft/server/DataConverterRegistry.java
++++ b/src/main/java/net/minecraft/server/DataConverterRegistry.java
+@@ -18,7 +18,8 @@ public class DataConverterRegistry {
+     // Yatopia start
+     private static final DataFixer c;
+     static {
+-        DataFixerBuilder datafixerbuilder = new DataFixerBuilder(SharedConstants.getGameVersion().getWorldVersion());
++        // DataFixerBuilder datafixerbuilder = new DataFixerBuilder(SharedConstants.getGameVersion().getWorldVersion());
++        DataFixerBuilder datafixerbuilder = new me.steinborn.lazydfu.mod.LazyDataFixerBuilder(SharedConstants.getGameVersion().getWorldVersion());
+ 
```

Comment:
Remove commented out line and add //Yatopia comment

Reply:
Fixed.